docs/content/guides/2.alokai-essentials/1.alokai-next-js/6.product-page.md

```diff
@@ -0,0 +1,358 @@
+---
+title: Build Product Details Page
+layout: default
+navigation:
+  icon: tabler:number-5-small
+---
+
+# Build Product Details Page with Storefront UI
+
+Every eCommerce application needs appealing, fast and responsive UI to attract and retain customers. With Storefront UI building one is easy and fast! In this guide, let's learn how to build a Product Details page. We will learn how to use Storefront UI components, blocks and of course how to customise them.
+
+::info
+Already familiar with **Storefront UI**? You can skip this section and jump to [Connecting Product Details Page with SAP Commerce Cloud](./6.connecting-pdp). You can find the code for this guide in the [nextjs-starter/product-page branch](https://github.com/vuestorefront-community/nextjs-starter/tree/product-page)
+::
+
+## Installing Storefront UI 
+
+First, in order to use Storefront UI we need to install and configure it. Luckily, it's very easy to do.
+
+### Install Storefront UI
+
+In the `storefront` directory, install Storefront UI by running the following command:
+
+```bash
+npm i @storefront-ui/react
+npm i -D tailwindcss postcss autoprefixer
+```
+
+This will install Storefront UI package for React based applications and Tailwind CSS with PostCSS and Autoprefixer.
+
+Next, initialize Tailwind CSS by running the following command:
+
+```bash
+npx tailwindcss init -p
+```
+
+All set! Now, let's configure Storefront UI.
+
+### Configure Storefront UI
+
+In the `storefront` directory, rename the `tailwind.config.js` file to `tailwind.config.ts` and update the file with the following content:
+
+```typescript
+const { tailwindConfig } = require('@storefront-ui/react/tailwind-config');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  presets: [tailwindConfig],
+  content: [
+    './app/**/*.{js,ts,jsx,tsx}',
+    './components/**/*.{js,ts,jsx,tsx}',
+    '../../node_modules/@storefront-ui/react/**/*.{js,mjs}',
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+};
+```
+
+This will configure Tailwind CSS to work with Next.js App Router file structure and Storefront UI.
```

Comment:
```suggestion
This will configure Tailwind CSS to work with Next.js App Router file structure and Storefront UI. This also means that only the Tailwind classes from the Storefront UI components you use will end up in your final CSS.
```